app/assets/javascripts/app/views/feedback_view.js

```diff
@@ -40,7 +43,40 @@ app.views.Feedback = app.views.Base.extend({
     if(evt) { evt.preventDefault(); }
     if(!window.confirm(Diaspora.I18n.t("reshares.post", {name: this.model.reshareAuthor().name}))) { return }
     this.model.interactions.reshare();
-  }
+  },
+
+  blockUser: function(evt) {
+    if(evt) { evt.preventDefault(); }
+    if(!confirm(Diaspora.I18n.t('ignore_user'))) { return; }
+
+    this.model.blockAuthor()
+      .done(function() {
+        // return to stream
+        document.location.href = "/stream";
```

Comment:
I guess there is no way to not hardcode links into the JS files... tried searching for existing ones, couldn't seem to figure out if we have anything. It would be nice to push all app paths to gon at some point, but that is probably not in scope of this.